package.json

```
@@ -51,6 +51,7 @@
     "nyc": "15.0.1",
     "proxyquire": "2.1.3",
     "semantic-release": "17.0.6",
+    "statuspage": "^0.1.6",
```

```suggestion
    "statuspage": "0.1.6",
```

other dependencies are strict...